### DISCUSSION TITLE
`set unnamedplus` is causing issues

### DISCUSSION THREAD

**Discussion by txtyash** [LazyVim/LazyVim#934] [General]
This line https://github.com/LazyVim/LazyVim/blob/2e7ad2b8257b7d25df0264a5b193da7af35f5a53/lua/lazyvim/config/options.lua#L8
causes issues by setting the default clipboard to the system clipboard. I don't think anyone would like that as a default. Currently I'm setting an option in my settings keep the clipboards separate:
```lua
opt.clipboard = "" -- Sync with system clipboard
```

**Comment by dpetka2001** [comment]
Well I for once, and I'm sure others too, like this feature. I would set it in my personal config if it wasn't enabled by default. Just like you did to disable it. It's just a matter of personal preference.

**Reply by txtyash** [reply]
Actually for most users who come from vim/neovim might not be used to this. Like me I had not idea about `set clipboard=unnamedplus` and thought that it was an issue, which had me wandering. I think it is confusing to change the basic settings like this...

**Reply by folke** [reply]
It's a personal preference for sure. And I'll keep it like this.

**Reply by nnutter** [reply]
While it is a personal preference it's a terrible experience for someone who uses a clipboard manager; which many more macOS users may now have because Spotlight can index clipboard history on macOS 26. It's a terrible experience because it not only applies to yanks but also deletes/changes, which are much more common than yanks, which means the clipboard history is flooded with garbage. I'll disable it but you should re-consider this. It took me quite awhile to even realize it was LazyVim which was causing the problem because it's "spooky action at a distance".
